styles.css

```diff
@@ -0,0 +1,231 @@
+@media (max-width: 989px) {
+ article {
+    width: 220px;
+    height: 300px;
+    background-color: #A2C4C9;
+    float: left;
+    border: 3px solid black;
+    border-radius: 5px;    
+    margin: 6px;
+    margin-top: 80px;
+    font-size: x-small;
+    box-sizing: border-box;
+    padding: 10px
+    }
+  #first {
+    height: 300px;
+    width: 400px
+  }
+  #second {
+    height: 190px;
+    width: 310px
+  }
+  #third {
+    height: 170px;
+    position: relative;
+    bottom: 75px;
+    width: 700px;
+    
+  }
+  article h1 {
+    text-align: center;
+    font-size: large;
+    margin-top: -6px;
+  }
+  article h3 {
+    text-align: center;
+    font-size: small;
+    margin-top: -14px
+  }
+  article h2 {
+    font-size: small;
+  }
+  header {
+    height: 75px;
+    border-bottom: 2px solid black;
+    margin-left: 25px;
+    background-color: #B6D7A8;
+    position: fixed;
+    top: 0px;
+    width: 100%
+  }
+  header h1 {
+    font-size: large;
+    text-align: center
+  }
+  header h2 {
+    font-size: small;
+    text-align: center;
+    margin-top: -9px
+  }
+  nav {
+    background-color: #D5A6BD;
+    width: 0px;
+    height: 400px;
+    position: absolute;
+    border: 2px solid black;
+    text-align: center;
+    font-size: small;
+    visibility: hidden

```

Comment:
You should be able to delete everything else about <nav> (the ad section) except for "visibility: hidden;".